test/system/navbar_links_test.rb

```diff
@@ -48,7 +48,7 @@ class NavbarLinksTest < ApplicationSystemTestCase
         @user2 = create :user, role: :data_volunteer
         log_in_as @user2
         visit authenticated_root_path
-        click_link t('navigation.admin_tools.label')
+        click_button t('navigation.admin_tools.label')
       end
```

Comment:
If we want to future proof these a little, we could use 'click_on' which works with both links and buttons

Reply:
this is a great call -- will sub!